client/components/Modal/PageCreateModal.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PageCreateModal from './PageCreateModal'
+import Crowi from '../../util/Crowi'
+import i18n from '../../i18n'
+
+i18n()
+
+const crowi = new Crowi({ user: {}, csrfToken: '' }, window)
```

Comment:
It does not work well😭

Reply:
Probably modal errors are not displayed in my environment…
Can I check your error?

Reply:
No errors are displayed, but user name isn't also displayed.

Reply:
OK! I added user name like this:

https://github.com/crowi/crowi/blob/3353e7005e5de3913dc960e1cbeefac48f5da0cc/client/components/Modal/PageCreateModal.stories.tsx#L8

<img width="513" alt="Screen Shot 2019-10-26 at 22 43 03" src="https://user-images.githubusercontent.com/38908416/67620459-006a3080-f842-11e9-8cee-d8e32140b95e.png">